src/qibo/quantum_info/clifford.py

```diff
@@ -150,6 +152,8 @@ def stabilizers(self, symplectic: bool = False, return_array: bool = False):
             return_array (bool, optional): To be used when ``symplectic = False``.
                 If ``True`` returns the stabilizers as ``ndarray``.
                 If ``False``, returns stabilizers as strings. Defaults to ``False``.
+            cores (int, optional): number of processes to be used to calculate all stabilizers.
```

Comment:
I think I would rather call this `ncpus` or similar, which from my experience is a more commonly used name for this kind of things.